podcasts/SwitchCell.swift

```diff
@@ -1,14 +1,12 @@
 import UIKit
 
 class SwitchCell: ThemeableCell {
-    @IBOutlet var cellSwitch: ThemeableSwitch!
+    let cellSwitch: ThemeableSwitch = {
+        let cellSwitch = ThemeableSwitch()
+        cellSwitch.isAccessibilityElement = false
```

Comment:
I believe you don't need to set it to false, when you add it to `accessoryView` it should work as expected

Reply:
Not realy, it gets exposed as accessibility element along side parent cell. I tested it manually and needed to set it to false.